Docs/source/install/hpc/summit.rst

```diff
@@ -41,7 +41,7 @@ We use the following modules and environments on the system (``$HOME/warpx.profi
    # required dependencies
    module load cmake/3.20.2
    module load gcc/9.3.0
-   module load cuda/11.0.3
+   module load cuda/11.3.1
```

Comment:
See the comment about not going past 11.1

Reply:
Oops, looks like the comment was edited. Sorry for the noise

Reply:
Commend extended, it's good - AMReX devs only observed issues with 11.4 so far